invenio_circulation/api.py

```diff
@@ -53,6 +51,16 @@ def get_record_by_pid(cls, pid, with_deleted=False):
         persistent_identifier, record = resolver.resolve(str(pid))
         return record
 
+    def attach_item_ref(self, data):
+        """Attach item reference."""
+        item_ref_builder = current_app.config.get(
+            "CIRCULATION_ITEM_REF_BUILDER")
+        item_pid = data.get("item_pid")
+        if item_ref_builder and item_pid:
+            self["item_pid"] = item_pid
+            self["item"] = item_ref_builder(self['loan_pid'])
+        return self
```

Comment:
I am not sure this is needed as we mutate the instance already :)

Reply:
ah yes, a leftover from copying